lib/mix/tasks/openapi.json.ex

```diff
@@ -0,0 +1,97 @@
+defmodule Mix.Tasks.Openapi.Spec.Json do
+  @moduledoc """
+  Serialize the given OpenApi spec module to a json file.
+  """
+  use Mix.Task
+  require Mix.Generator
+
+  @default_filename "openapi.json"
```

Comment:
Since the output file name is not critical, I think having a good default value is effective in terms of UX. In addition, many projects and official documentation also recommend this name:

```
It is RECOMMENDED that the root OpenAPI document be named: openapi.json or openapi.yaml
```